kubernetes/helm/ckeditor-cs-development-stack/README.md

```diff
@@ -0,0 +1,70 @@
+# CKEditor Collaboration Server On-Premises development stack Helm chart
+
+>:warning: **Not for production purposes!** This configuration is insecure, not
+>persistent and its purpose is testing only.
+
+Helm chart for fast deployment of Collaboration Server On-Premises in Kubernetes
+with MySQL database and Redis cluster.
+
+## Minimum requirements
+- 4 CPU Core
+- 4096MB RAM
+- Kubernetes 1.19+
+- Helm v3
+
+## Quick start
+
+Create imagePullSecret for pulling images from CKEditor container registry,
+replace `xxx` with authentication token
+```sh
+kubectl create secret docker-registry docker-cke-cs-com \
+    --docker-username "cs" \
+    --docker-server "https://docker.cke-cs.com" \
+    --docker-password="xxx"
+```
+
+Installing helm chart in cluster:
+```sh
+cd ckeditor-cs-development-stack
+helm repo add bitnami https://charts.bitnami.com/bitnami
+helm repo update
+helm dependency update
+helm install ckeditor-cs .
+```
+
+Deleting it:
+```sh
+helm delete ckeditor-cs
+```
+## Local environment
```

Comment:
It looks the container is running for me, but I'm :baby:  in K8 and I don't know how I can check it manually - I don't know URL for running instance.   
  
And will be great to have a section with instruction on how you can test it. We can use approach with running tests `kubectl run ckeditor-cs-tests -i --rm \`

Reply:
Added a separate file with script running e2e tests. As for URL, there is now information about default hostname from which collaboration server should be accessible by default.